packages/nc-gui/pages/[projectType]/[projectId]/index.vue

```diff
@@ -510,12 +515,16 @@ onBeforeUnmount(reset)
           </div>
         </div>
 
-        <LazyDashboardTreeView />
+        <LazyDashboardTreeView @create-base-dlg="toggleDialog(true, 'dataSources')" />
       </a-layout-sider>
     </template>
 
-    <div>
-      <LazyDashboardSettingsModal v-model="dialogOpen" :open-key="openDialogKey" />
+    <div :key="$route.fullPath.split('?')[0]">
```

Comment:
Any reason to provide the key?  this will re-render whenever we switch between tables/views.